terraform/README.md

```diff
@@ -81,3 +80,34 @@ The following steps need to be followed:
 
 
 More Details to Variables and Locals can be found in the template README´s
+
+## Customizing the deployment further
+
+To see a full list of variables that can be changed to customize the deployment see:
+
+Infrastructure part: [https://github.com/sotec-iot/hono-extras/blob/terraform/terraform/infrastructure/variables.tf](https://github.com/sotec-iot/hono-extras/blob/terraform/terraform/software/variables.tf)<br/>
+Software part: [https://github.com/sotec-iot/hono-extras/blob/terraform/terraform/software/variables.tf](https://github.com/sotec-iot/hono-extras/blob/terraform/terraform/software/variables.tf)
+
+### Changing a variable
+To change the value of a variable, e.g. grafana_expose_externally, add the variable in the corresponding locals.tf file and assign it a new value.<br/>
+For our example:
+```
+grafana_expose_externally = true
+```
+Now go to the corresponding main.tf file and add the following line
+```
+VARIABLE = local.VARIABLE
+```
+Replace `VARIABLE` with the name of the variable you want to change.<br/>
+For our example:
+```
+grafana_expose_externally = local.grafana_expose_externally
+```
+
+### Changing an object variable
+To change the properties within a variable of type object you have to follow the same steps as for a simple variable (described above) except you have to provide an object in the locals.tf file.
+For object variables that have optional properties you can just provide the non-optional properties and the ones you want to change.<br/>
```

Comment:
```suggestion
For object variables that have optional properties, provide the non-optional properties and the ones you want to change.<br/>
```

Reply:
Sentence changed to: "For object variables that have optional properties, it is sufficient to provide only the non-optional properties and the properties you want to change."